src/dotnet/commands/dotnet-test/LocalizableStrings.cs

```diff
@@ -52,5 +52,7 @@ internal class LocalizableStrings
                                         Logs are written to the provided file.";
 
         public const string CmdNoBuildDescription = @"Do not build project before testing.";
+
+        public const string RunSettingsArgsHelpText = @"Any extra options that should be passed to MSBuild or adapter. See 'dotnet msbuild -h' and 'dotnet vstest --help' for available options.";
```

Comment:
I was surprised that a different help switch appears to be needed for these two commands. Is this by design? @livarcocc @piotrpMSFT